.github/workflows/cypress.yml

```
@@ -104,11 +111,10 @@ jobs:
         uses: cypress-io/github-action@59810ebfa5a5ac6fcfdcfdf036d1cd4d083a88f2 # v6.5.0
         with:
           component: ${{ matrix.containers == 'component' }}
-          group: ${{ matrix.use-cypress-cloud && matrix.containers == 'component' && 'Run component' || matrix.use-cypress-cloud && 'Run E2E' || '' }}
-          # cypress env
-          ci-build-id: ${{ matrix.use-cypress-cloud && format('{0}-{1}', github.sha, github.run_number) || '' }}
-          tag: ${{ matrix.use-cypress-cloud && github.event_name || '' }}
```

This is not related

---

But invalid as there is no `matrix.use-cypress-cloud`

---

Right, but the group, ci build and tag are still mandatory and relevant to have

---

Well as the variable does not exist all of them are set to ''
So it makes no difference

---

Doesn't mean we should dro pit instead of fixing it :thinking: 

- https://github.com/nextcloud/server/pull/40736 changed the behaviour
- https://github.com/nextcloud/server/pull/41039 changed it too but broke the group/tag 

I'd suggest we revert back to

```suggestion
          group: Run ${{ matrix.containers == 'component' && 'component' || 'E2E' }}
          # cypress env
          ci-build-id: ${{ github.sha }}-${{ github.run_number }}
          tag: ${{ github.event_name }}
```

---

makes sense

---

Well this breaks the CI, as those flags are only allowed when using the cypress cloud which we do not.

ref: https://docs.cypress.io/guides/references/error-messages#The---ci-build-id---group---tag---parallel-or---auto-cancel-after-failures-flags-can-only-be-used-when-recording

---

Time for our custom cloud :cry:

---

https://github.com/nextcloud/server/commits/39158c0a52c478df989c955603c643f3905a7343